grpc-server/src/main/scala/com/wavesplatform/api/grpc/AccountsApiGrpcImpl.scala

```diff
@@ -65,4 +68,18 @@ class AccountsApiGrpcImpl(blockchain: Blockchain)(implicit sc: Scheduler)
 
     responseObserver.completeWith(stream)
   }
+
+  override def resolveAlias(request: StringValue): Future[BytesValue] =
+    Future {
+      val addressEither = for {
+        alias   <- Alias.create(request.value)
+        address <- blockchain.resolveAlias(alias)
+      } yield BytesValue(address.bytes)
+
+      Future.fromTry {
+        addressEither
+          .leftMap(_.toException)
+          .toTry
+      }
+    }.flatten
```

Comment:
1. Please remove `ValidationErrorException`
1. Replace `Future.fromTry` with `addressEither.explicitGet()`

Reply:
`ValidationErrorException` required for a gRPC errors logic, because plain Exceptions from `explicitGet()` omits the corresponding ValidationError, so we cant embed it conveniently into a gRPC status response.